src/main/java/spireCafe/interactables/CafeRoom.java

```diff
@@ -46,43 +78,26 @@ public void onEnterRoom() {
         List<Class<? extends AbstractCafeInteractable>> possibleAttractions = getPossibilities(AbstractAttraction.class);
 
         Collections.shuffle(possibleBartenders, new java.util.Random(rng.randomLong()));
-        this.bartender = (AbstractBartender) createInteractable(possibleBartenders.get(0), 800*Settings.xScale, AbstractDungeon.floorY+400*Settings.yScale);
+        this.bartender = (AbstractBartender) createInteractable(possibleBartenders.get(0), 1200 * Settings.xScale, AbstractDungeon.floorY + 100 * Settings.yScale);
         Anniv7Mod.currentRunSeenInteractables.add(bartender.id);
 
         int numPatrons = 3;
         Collections.shuffle(possiblePatrons, new java.util.Random(rng.randomLong()));
         for (int i = 0; i < numPatrons && i < possiblePatrons.size(); i++) {
             float x = (1000 + i * 200.0f) * Settings.xScale;
             float y = AbstractDungeon.floorY;
-            AbstractNPC patron = (AbstractNPC)createInteractable(possiblePatrons.get(i), x, y);
+            AbstractNPC patron = (AbstractNPC) createInteractable(possiblePatrons.get(i), x, y);
             this.npcs.add(patron);
             Anniv7Mod.currentRunSeenInteractables.add(patron.id);
         }
 
         Collections.shuffle(possibleMerchants, new java.util.Random(rng.randomLong()));
         //this.merchant = (AbstractMerchant)createInteractable(possibleMerchants.get(0), 1000*Settings.xScale, AbstractDungeon.floorY+400*Settings.yScale);
-        this.merchant = (AbstractMerchant)createInteractable(SnackmasterMerchant.class, 1000*Settings.xScale, AbstractDungeon.floorY+400*Settings.yScale);
+        this.merchant = (AbstractMerchant) createInteractable(SnackmasterMerchant.class, 200 * Settings.xScale, AbstractDungeon.floorY);
```

Comment:
Can probably remove the snackmaster and put back the commented out code from above